### ISSUE TITLE
Constructors wrongly referenced in demo

### ISSUE THREAD

**Issue by facundoj** [IMSGlobal/caliper-js-public#1]
Some constructors are not being referenced correctly from the demo file. For example:
In _index.html_ 

``` js
var actor = new Caliper.Person("https://example.edu/user/554433");
```

should be:

``` js
var actor = new Caliper.Entities.Person("https://example.edu/user/554433");
```

It is inconsistent with the [sample application](http://plnkr.co/edit/6uRssC6aGXdXWK4l1UjX?p=info) in Best Practice Guide.

**Comment by arwhyte** [comment]
Yep, the index.html contains bad references.  Will correct.  Thanks for reporting.

**Comment by scody** [comment]
I think this is still a problem?

**Comment by arwhyte** [comment]
We've fixed this for 1.1.  We've yet to decide if we plan to release a 1.0.1 with this as a fix.

**Comment by arwhyte** [comment]
Closing as 1.1 has been released.
